create-test-user.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable no-console */
+const config = require('config');
+const hoek = require('hoek');
+
+// Setup Authentication
+const authConfig = config.get('auth');
+
+// Setup HTTPd
+const httpdConfig = config.get('httpd');
+
+// Special urls for things like the UI
+const ecosystem = config.get('ecosystem');
+
+ecosystem.api = httpdConfig.uri;
+
+// Setup Datastore
+const datastoreConfig = config.get('datastore');
+const DatastorePlugin = require(`screwdriver-datastore-${datastoreConfig.plugin}`);
+const datastore = new DatastorePlugin(hoek.applyToDefaults({ ecosystem },
+    (datastoreConfig[datastoreConfig.plugin] || {})));
+
+// Source Code Plugin
+const scmConfig = config.get('scm');
+const ScmPlugin = require(`screwdriver-scm-${scmConfig.plugin}`);
+const scm = new ScmPlugin(hoek.applyToDefaults({ ecosystem },
+    (scmConfig[scmConfig.plugin] || {})));
+
+authConfig.scm = scm;
+
+// Setup Model Factories
+const Models = require('screwdriver-models');
+const userFactory = Models.UserFactory.getInstance({
+    datastore,
+    scm,
+    password: authConfig.encryptionPassword
+});
+const tokenFactory = Models.TokenFactory.getInstance({
+    datastore
+});
+
+const username = process.argv[2];
+const gitToken = process.argv[3];
```

Comment:
Oh whoops I used to have a usage block, must have missed that when I moved it from a standalone to part of the SD repo!